specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-04-01-preview/migrate.json

```diff
@@ -0,0 +1,15310 @@
+          },
+          {
+            "name": "StandardE16V4",
+            "value": "Standard_E16_v4"
+          },
+          {
+            "name": "StandardE20V4",
+            "value": "Standard_E20_v4"
+          },
+          {
+            "name": "StandardE32V4",
+            "value": "Standard_E32_v4"
+          },
+          {
+            "name": "StandardE48V4",
+            "value": "Standard_E48_v4"
+          },
+          {
+            "name": "StandardE64V4",
+            "value": "Standard_E64_v4"
+          },
+          {
+            "name": "StandardE2SV4",
+            "value": "Standard_E2s_v4"
+          },
+          {
+            "name": "StandardE4SV4",
+            "value": "Standard_E4s_v4"
+          },
+          {
+            "name": "StandardE8SV4",
+            "value": "Standard_E8s_v4"
+          },
+          {
+            "name": "StandardE16SV4",
+            "value": "Standard_E16s_v4"
+          },
+          {
+            "name": "StandardE20SV4",
+            "value": "Standard_E20s_v4"
+          },
+          {
+            "name": "StandardE32SV4",
+            "value": "Standard_E32s_v4"
+          },
+          {
+            "name": "StandardE48SV4",
+            "value": "Standard_E48s_v4"
+          },
+          {
+            "name": "StandardE64SV4",
+            "value": "Standard_E64s_v4"
+          },
+          {
+            "name": "StandardE2DV4",
+            "value": "Standard_E2d_v4"
+          },
+          {
+            "name": "StandardE4DV4",
+            "value": "Standard_E4d_v4"
+          },
+          {
+            "name": "StandardE8DV4",
+            "value": "Standard_E8d_v4"
+          },
+          {
+            "name": "StandardE16DV4",
+            "value": "Standard_E16d_v4"
+          },
+          {
+            "name": "StandardE20DV4",
+            "value": "Standard_E20d_v4"
+          },
+          {
+            "name": "StandardE32DV4",
+            "value": "Standard_E32d_v4"
+          },
+          {
+            "name": "StandardE48DV4",
+            "value": "Standard_E48d_v4"
+          },
+          {
+            "name": "StandardE64DV4",
+            "value": "Standard_E64d_v4"
+          },
+          {
+            "name": "StandardE2DsV4",
+            "value": "Standard_E2ds_v4"
+          },
+          {
+            "name": "StandardE4DsV4",
+            "value": "Standard_E4ds_v4"
+          },
+          {
+            "name": "StandardE8DsV4",
+            "value": "Standard_E8ds_v4"
+          },
+          {
+            "name": "StandardE16DsV4",
+            "value": "Standard_E16ds_v4"
+          },
+          {
+            "name": "StandardE20DsV4",
+            "value": "Standard_E20ds_v4"
+          },
+          {
+            "name": "StandardE32DsV4",
+            "value": "Standard_E32ds_v4"
+          },
+          {
+            "name": "StandardE48DsV4",
+            "value": "Standard_E4
```

Comment:
Check all field descriptions in this object

Reply:
I think description is missing because of a lot of possible values for this enum.
can you please confirm if its required (must add) or recommended (can be skipped) by ARM to add descriptions for all values of enums or models in the generated swagger?
 
@tanvidhope-microsoft for taking the change.